encord/client.py

```diff
@@ -1295,16 +1300,25 @@ def __set_project_ontology(self, ontology: Ontology) -> bool:
         payload = {"editor": ontology.to_dict()}
         return self._querier.basic_setter(OrmProject, uid=None, payload=payload)
 
-    def workflow_reopen(self, label_hashes: list[str]) -> None:
+    def workflow_reopen(self, label_hashes: List[str]) -> None:
```

Comment:
yep. `list` only works in newer pythons :(

Reply:
I'm surprised that our SDK integration test pipelines didn't catch that. It should have failed when it read this code for older Python versions. @sergei-encord do you know what happened there?

Reply:
My question was more around: why the SDK integration tests did not raise any runtime errors here because of the wrong syntax?

Reply:
why would it? it is only type annotation, python totally ignores them in runtime

Reply:
It should error out. Try it! Somehow we're not caching failing tests.
```py
╭─denis@MacBook-Pro-2 ~/Documents/other-repos
╰─$ python3.7
Python 3.7.16 (default, Dec  7 2022, 01:34:15)
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def bla(x: list[str]):
...     pass
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
>>> from typing import List
>>>
>>> def bla(x: List[str]):
...     pass
...
>>>

```

Reply:
Ah, interesting, I seen it not being critical, perhaps I just mistook wrong python version 🤕